### PR TITLE
[Redesign] Contact Support page

### PR DESCRIPTION
**Pull request by scottbommarito**
image

Couple notes:
- Fixed some broken Javascript validation behavior by copying some code from the old common Javascript file (`nugetgallery.js`) to `common.js`
- Made some minor unifications to the `Report Abuse` page, most notably resizing to 10 columns.

image

### CODE REVIEW

File: src/NuGetGallery/Views/Packages/ReportMyPackage.cshtml [hunk 1]
```diff
@@ -2,55 +2,55 @@
...
-    </fieldset>
-}
+    </div>
+</div>
 
 @section BottomScripts {
-    <script src="@Url.Content("~/Scripts/jquery.validate.min.js")" type="text/javascript"></script>
-    <script src="@Url.Content("~/Scripts/jquery.validate.unobtrusive.min.js")" type="text/javascript"></script>
     @Html.SpamPreventionScript()
+    <script>
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Packages/ReportMyPackage.cshtml]: type="text/javascript"

File: src/NuGetGallery/Views/Packages/ReportAbuse.cshtml [hunk 2]
```diff
@@ -7,33 +7,32 @@
...
 
 <section class="container main-container page-report-abuse">
-    <h1 class="page-heading">Report Abuse</h1>
+    <div class="row report-form">
+        <div class="col-md-offset-1 col-md-10">
+            <h1 class="page-heading text-center">Report Abuse by <a href="@Url.Package(@Model.PackageId, Model.PackageVersion)">@Model.PackageId @Model.PackageVersion</a></h1>
 
-    @ViewHelpers.AlertWarning(isAlertRole: true, htmlContent:
+            @ViewHelpers.AlertWarning(isAlertRole: true, htmlContent:
         @<text>
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Packages/ReportAbuse.cshtml]: It looks like the @<text>...</text>) isn't properly aligned

File: src/NuGetGallery/Views/Packages/ReportMyPackage.cshtml [hunk 3]
```diff
@@ -2,55 +2,55 @@
...
+        function reasonSelected() {
+            var $form = $('#Reason').parents('form');
+            var val = $('#Reason').val();
+            if (val) {
+                $form.validate().element($('#Reason'));
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Packages/ReportMyPackage.cshtml]: nit: you could move the $form selector on line 44 to inside this conditional

File: src/NuGetGallery/Views/Packages/ReportAbuse.cshtml [hunk 4]
```diff
@@ -7,33 +7,32 @@
...
-
-    @if (!Model.ConfirmedUser)
-    {
-        <p>
-            Note: if you are the owner of @Model.PackageId, please
-            <a href="@Url.LogOn(returnUrl)">sign in now before you complete this form.</a>
-        </p>
-    }
+            <p>
+                Please provide a detailed abuse report. Include exactly what the package did, for example if you are reporting a package with a virus or malicious code,
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Packages/ReportAbuse.cshtml]: This message seems awkward. Would something along these lines be better?

> Please provide a detailed abuse report with evidence to support your claim! We cannot delete packages without evidence that they exhibit malicious behavior.

File: src/NuGetGallery/Views/Packages/ReportAbuse.cshtml [hunk 5]
```diff
@@ -46,6 +45,7 @@
                 </div>
                 <div id="form-field-reason" class="form-group @Html.HasErrorFor(m => m.Reason)">
                     @Html.ShowLabelFor(m => m.Reason)
+                    <p>Please select the reason for contacting support about your package.</p>
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Packages/ReportAbuse.cshtml]: This is the report abuse page, this message should be: "Please select the reason for contacting support about this package".

File: src/NuGetGallery/Views/Packages/ReportAbuse.cshtml [hunk 6]
```diff
@@ -56,7 +56,7 @@
                 </div>
                 <div class="form-group @Html.HasErrorFor(m => m.Message)">
                     @Html.ShowLabelFor(m => m.Message)
-                    <p>In addition to selecting the reason for reporting the package, you must provide details of the problem here.</p>
+                    <p>Please provide a detailed description of your problem.</p>
```
Comment (loic-sharma) [inline src/NuGetGallery/Views/Packages/ReportAbuse.cshtml]: nit: I think "the" instead of "your" comes off as a little less aggressive.

**Pull request merged**